cinema/serializers.py

```diff
@@ -69,13 +95,15 @@ class MovieSessionListSerializer(MovieSessionSerializer):
         source="cinema_hall.capacity", read_only=True
     )
     tickets_available = serializers.IntegerField(read_only=True)
+    movie_image = serializers.ImageField(source="movie.image", read_only=True)
```

Comment:
Good job on using `serializers.ImageField` for the `movie_image` field as per the checklist. This is more efficient than using `serializers.CharField`.